backend/README.md

```diff
@@ -0,0 +1,49 @@
+## Run locally
+
+You'll need access to the Firebase project at https://console.firebase.google.com/project/c19-backend.
+
+### Sign into Firebase
+
+From the `backend` directory:
+
+```
+npm install -g firebase-tools
+firebase login
```

Comment:
Does `firebase-tool` need to be installed globally or can we include it in our `package.json`.  If so, I'm assuming we could also just add an entry to the `scripts` to map `yarn login` to `firebase login`

Reply:
I think it does need to be installed globally so the `firebase` command is available, but I'm not certain.